filesystem/src/main/java/com/nytimes/android/external/fs/filesystem/FSFile.java

```diff
@@ -71,6 +71,9 @@ public void write(BufferedSource source) throws IOException {
             if (sink != null) {
                 sink.close();
             }
+            if (source != null) {
+                source.close();
+            }
```

Comment:
@PaulWoitaschek That's a fair point. Annotated the arg and removed the null check.